wp-content/plugins/core/src/Blocks/Types/Content_Loop/Content_Loop.php

```diff
@@ -0,0 +1,325 @@
+	public const MANUAL_POST_MESSAGE = 'manual_post_message';
+	public const MANUAL_POST         = 'manual_post';
+	public const MANUAL_TOGGLE       = 'manual_toggle';
+	public const MANUAL_TITLE        = 'manual_title';
+	public const MANUAL_EXCERPT      = 'manual_excerpt';
+	public const MANUAL_CTA          = 'manual_cta';
+	public const MANUAL_THUMBNAIL    = 'manual_thumbnail';
+	public const MANUAL_UPPER_META   = 'manual_upper_meta';
+	public const MANUAL_LOWER_META   = 'manual_lower_meta';
+
+	//Query Fields
+	public const QUERY_GROUP      = 'query_group';
+	public const QUERY_LIMIT      = 'query_limit';
+	public const QUERY_TAXONOMIES = 'query_taxonomy_terms';
+	public const QUERY_POST_TYPES = 'query_post_types';
+
+	//Layout Fields
+	public const LAYOUT           = 'layout';
+	public const LAYOUT_ROW       = 'layout_row';
+	public const LAYOUT_FEATURE   = 'layout_feature';
+	public const LAYOUT_COLUMNS   = 'layout_columns';
+
+	public const ALLOWED_POST_TYPES = [
+		Post::NAME,
+		Page::NAME,
+	];
+
+	public function add_block() {
+		$this->set_block( new Block( self::NAME, [
+			'title'       => __( 'Content Loop', 'tribe' ),
+			'description' => __( 'A loop of auto or manual set posts with style options', 'tribe' ),
+			'icon'        => '<svg enable-background="new 0 0 146.3 106.3" version="1.1" viewBox="0 0 146.3 106.3" xml:space="preserve" xmlns="http://www.w3.org/2000/svg"><style type="text/css">.st0{fill:#16D690;}.st1{fill:#21A6CB;}.st2{fill:#008F8F;}</style><polygon class="st0" points="145.2 106.3 72.6 42.3 26.5 1.2 0 106.3"/><polygon class="st1" points="145.2 106.3 0 106.3 72.6 42.3 118.6 1.2"/><polygon class="st2" points="72.6 42.3 145.2 106.3 0 106.3"/></svg>', // TODO: set SVG icon
+			'keywords'    => [ __( 'content', 'loop', 'tribe' ) ],
+			'category'    => 'common',
+			'supports'    => [
+				'align'  => false,
+				'anchor' => true,
+				'html'   => false,
+			],
+		] ) );
+	}
+
+	/**
+	 * Register Fields for block
+	 */
+	public function add_fields() {
+		//==========================================
+		// Content Fields
+		//==========================================
+		$this->add_section( new Field_Section( self::SECTION_CONTENT, __( 'Content', 'tribe' ), 'accordion' ) )
+			->add_field( new Field( self::NAME . '_' . self::TITLE, [
+				'label' => __( 'Title', 'tribe' ),
+				'name'  => self::TITLE,
+				'type'  => 'text',
+				] )
+			)->add_field( new Field( self::NAME . '_' . self::LEADIN, [
+					'label' => __( 'Lead in', 'tribe' ),
+					'name'  => self::LEADIN,
+					'type'  => 'text',
+				] )
+			)->add_field( new Field( self::NAME . '_' . self::DESCRIPTION, [
+					'label' => __( 'Description', 'tribe' ),
+					'name'  => self::DESCRIPTION,
+					'type'  => 'wysiwyg',
+				] )
+			)->add_field( new Field( self::NAME . '_' . self::CTA, [
+					'label' => __( 'Call to Action', 'tribe' ),
+					'name'  => self::CTA,
+					'type'  => 'link',
+				] )
+			)->add_field( new Field( self::NAME . '_' . self::QUERY_TYPE, [
```

Comment:
I wonder if we should try to land the ACF query field work Mike did as part of this block. Thoughts, @ryanurban ?

Reply:
I'm going to let someone figure that out in another branch/ticket